website/tosti/settings/development.py

```diff
@@ -42,7 +42,8 @@
 MEDIA_URL = "/media/"
 
 # SPOTIFY SETTINGS
-SPOTIFY_CACHE_PATH = os.path.join(BASE_DIR, "cache")  # noqa
+SPOTIFY_CACHE_PATH = os.path.join(BASE_DIR, "cache/spotify")  # noqa
```

Comment:
Shouldn't we now have a marietje cache path too? And how should this be on production?